python/ray/scripts/scripts.py

```diff
@@ -799,8 +801,8 @@ def stop(force, verbose, log_style, log_color):
             cli_logger.warning("Try running the command again, or use `{}`.",
                                cf.bold("--force"))
 
-    # TODO(maximsmol): we should probably block until the processes actually
-    # all died somehow
+    # Wait for the processes to actually stop.
+    psutil.wait_procs(stopped, timeout=2)
```

Comment:
Great fix!!!!
Can we catch this and suggest that the user retries with `--force`?